utility_scripts/augment_classes_v2.py

```diff
@@ -110,7 +126,7 @@ def augment_for_multiple_classes(args):
     parser.add_argument("--augmented_image_dir", default='/home/trashwheel/annotated_images/', help="Directory to save augmented images.")
     parser.add_argument("--updated_annotations_file", default='/home/trashwheel-annotations/all_annotations.json', help="File path to save the updated annotations JSON.")
     parser.add_argument("--classes_to_augment", nargs='+', required=True, help="List of class names to augment.")
-    parser.add_argument("--grayscale", action='store_true', help="If set, converts images to grayscale as part of the augmentation process.")
+    parser.add_argument("--nighttime", action='store_true', help="If true, converts images to nighttime(Adds noise, grayscale, vignette).")
```

Comment:
update wording to use 'set' instead of 'true' here in the help statement. My fault here when I was telling you that this argument is set to true.